js_modules/dagster-ui/packages/ui-core/src/ticks/TickLogDialog.tsx

```diff
@@ -98,13 +101,61 @@ export const QueryfulTickLogsTable = ({instigationSelector, tick}: TickLogTableP
     return <TickLogsTable events={events} />;
   }
 
+  const tickStatus =
+    data?.instigationStateOrError.__typename === 'InstigationState'
+      ? data?.instigationStateOrError.tick.status
+      : undefined;
+  const instigationType =
+    data?.instigationStateOrError.__typename === 'InstigationState'
+      ? data?.instigationStateOrError.instigationType
+      : undefined;
+  const instigationLoggingDocsUrl =
+    instigationType === 'SENSOR'
+      ? 'https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#logging-in-sensors'
+      : instigationType === 'SCHEDULE'
+      ? 'https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#logging-in-schedules'
+      : undefined;
+
   return (
     <Box
       style={{height: 500}}
       flex={{justifyContent: 'center', alignItems: 'center'}}
       padding={{vertical: 48}}
     >
-      {loading ? 'Loading logs…' : 'No logs available'}
+      {loading ? (
+        'Loading logs…'
+      ) : (
+        <NonIdealState
+          icon="no-results"
+          title="No logs to display"
+          description={
+            <Box flex={{direction: 'column'}}>
+              <Body2 color={colorTextLight()} style={{padding: '6px 0'}}>
```

Comment:
- Does it look incorrect if the `colorTextLight()` isn't included here?
- Does it look incorrect if these are just `div` elements instead of `Body2`?

I wouldn't think that this `NonIdealState` needs its description text to look different from the default, so I'm just curious if these added styles are needed.

Reply:
Yeah it looks fine when using just `<div>` elements. Adding the `gap: 12` in the parent component helped. Thanks!